pkg/catalog/catalog.go

```diff
@@ -188,43 +119,18 @@ func ListDevfileComponents(registryName string) (DevfileComponentTypeList, error
 		return *catalogDevfileList, err
 	}
 
-	// 1. Load each devfile concurrently from the previously retrieved devfile index entries
-	// 2. Populate devfile components with devfile data
-	// 3. Add devfile component types to the catalog devfile list
-	retrieveDevfiles := util.NewConcurrentTasks(len(registryIndices))
-	devfileMutex := &sync.Mutex{}
+	// Add stacks to the catalog devfile list
 	for _, index := range registryIndices {
 		// Load the devfile
```

Comment:
Nit: we can remove this comment as we don't download/load the devfile.

Reply:
updated